src/super_gradients/recipes/training_hyperparams/coco2017_yolox_train_params.yaml

```diff
@@ -0,0 +1,55 @@
+defaults:
+  - default_train_params
+
+max_epochs: 300
+lr_mode: cosine
+cosine_final_lr_ratio: 0.05
+lr_warmup_epochs: 5
+lr_cooldown_epochs: 15
+warmup_bias_lr: 0.0
+warmup_momentum: 0.9
+initial_lr:  0.02
+warmup_mode: yolov5_warmup
+batch_accumulate: 1
+
+save_ckpt_epoch_list: [285]
+
+loss: yolox_loss
+
+criterion_params:
+  strides: [8, 16, 32]  # output strides of all yolo outputs
+  num_classes: 80
+
+loss_logging_items_names: [iou, obj, cls, l1, num_fg, Loss]
+
+optimizer: SGD
+optimizer_params:
+  momentum: 0.9
+  weight_decay: 0.0005  # will be scaled according to effective batch size
+  nesterov: True
+
+ema: True
+mixed_precision: True
+
+valid_metrics_list:
+  - _target_: super_gradients.training.metrics.DetectionMetrics
+    normalize_targets: True
+    post_prediction_callback:
+      _target_: super_gradients.training.models.detection_models.yolov5_base.YoloV5PostPredictionCallback
+      iou: 0.65
+      conf: 0.01
+    num_cls: 80
+
+pre_prediction_callback:
+  _target_: super_gradients.training.datasets.datasets_utils.DetectionMultiscalePrePredictionCallback
+
+phase_callbacks:
+  - _target_: super_gradients.training.utils.callbacks.YoloXTrainingStageSwitchCallback
+    next_stage_start_epoch: 285
```

Comment:
Can `save_ckpt_epoch_list` be interpolated from `phase_callbacks.0.next_stage_start_epoch`, or it won't work with a target? Not a must of course, but would be nice